cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -1359,9 +1370,6 @@
 ;; rely on something like `Inst::mov_mitosis` to put an operand into the
 ;; appropriate physical register for whatever reason.
 
```

Comment:
Should this comment also be deleted now? If I'm reading the surrounding context correctly, I think `xmm0` was the last thing the comment referred to.

Reply:
Yep, thanks for catching that!